sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_autoinstrumentation/configurator.py

```diff
@@ -26,6 +33,17 @@ def _configure(self, **kwargs):
         if not _is_attach_enabled():
             warn(_PREVIEW_ENTRY_POINT_WARNING)
         try:
+            if environ.get(OTEL_TRACES_EXPORTER, "").lower().strip() != "none":
+                kwargs.setdefault("trace_exporter_names", ["azure-monitor-opentelemetry-exporter"])
+                try:
+                    sample_rate = float(environ.get("OTEL_TRACES_SAMPLER_ARG"))
+                except:  # pylint: disable=bare-except
+                    sample_rate = 1.0
+                kwargs.setdefault("sampler", ApplicationInsightsSampler(sample_rate))
+            if environ.get(OTEL_METRICS_EXPORTER, "").lower().strip() != "none":
+                kwargs.setdefault("metric_exporter_names", ["azure-monitor-opentelemetry-exporter"])
```

Comment:
I believe this change opens up the possibility for duplicate azure monitor exporters to be added (one through the `kwargs` and then one if user specifies through env var `OTEL_X_EXPORTER`).

Reply:
Yeah. I am confirming with the other languages if that matches their implementations.

Reply:
Sent an email out. Note, though, that we do not tell customers anywhere to add exporters via these env vars. We only say that they can set them to "None" to disable the signal. That's still supported.
https://learn.microsoft.com/en-us/azure/azure-monitor/app/azure-web-apps-python#configuration

Reply:
Correct, however I can definitely see a future where we will be allowing customers to define custom exporters in the future via env vars. Since it is not currently documented, and auto-instrumentation is experimental (and only being used for attach), we can add a TODO for now.

Reply:
https://msazure.visualstudio.com/One/_workitems/edit/28312755